www/__tests__/timelineHelper.test.ts

```diff
@@ -0,0 +1,60 @@
+import { clearAlerts, mockAlert, mockLogger } from '../__mocks__/globalMocks';
+import { readAllCompositeTrips, readUnprocessedTrips } from '../js/diary/timelineHelper';
+import { mockBEMUserCache } from '../__mocks__/cordovaMocks';
+
+import * as mockTLH from '../__mocks__/timelineHelperMocks';
+
+mockLogger();
+mockAlert();
+mockBEMUserCache();
+
+beforeEach(() => {
+  clearAlerts();
+});
+
+afterAll(() => {
+  jest.restoreAllMocks();
+});
+
+// Once we have end-to-end testing, we could utilize getRawEnteries.
+jest.mock('../js/commHelper', () => ({
+  getRawEntries: jest.fn((key, startTs, endTs, valTwo) => {
+    if (startTs === mockTLH.fakeStartTsOne) return mockTLH.mockData;
+    if (startTs == mockTLH.fakeStartTsTwo) return mockTLH.mockDataTwo;
+    return {};
+  }),
```

Comment:
The approach is good, but it seems like `mockDataTwo` is just 2 of the same trip?